definitions/src/asm.rs

```diff
@@ -58,14 +60,22 @@ impl AsmCoreMachine {
         let mut machine = unsafe {
             let layout = Layout::new::<AsmCoreMachine>();
             #[allow(clippy::cast_ptr_alignment)]
-            // TODO: change this to alloc so we are using malloc instead of
+            // Use alloc so we are using malloc instead of
             // calloc, then do lazy zero filling when necessary. That might
             // save us some time in case a script doesn't use all the memory.
-            // Right now this calloc phase here takes around 1ms.
-            let raw_allocation = alloc_zeroed(layout) as *mut AsmCoreMachine;
+            let raw_allocation = alloc(layout) as *mut AsmCoreMachine;
             Box::from_raw(raw_allocation)
         };
```

Comment:
Now that we have no dependency on zeroed memory pages, maybe all we need here, is `Box::<AsmCoreMachine>::new()`?

Reply:
Maybe I didn't understand it correctly: I alloced memory from the OS, and then initialize `AsmCoreMachine` with zero value except `AsmCoreMachine.Memory`. The lazy memory initialization inside the vm will not affect here.

Reply:
I mean you can just probably initialize using Box, you don't need the manual unsafe allocation via layout module and alloc

Reply:
https://doc.rust-lang.org/std/boxed/struct.Box.html#impl

An `AsmCoreMachine` instance should be used in Box::new(x: T), this leads to the emergence of circular dependencies.